File: js/problem58.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    s = s.trim();
    if (s.length === 0) {
        return 0;
    }
    let charArr = s.split(' ');
    return charArr[charArr.length - 1].length;
};

File: leetcodeJava/src/main/java/Problem58.java
// 58. Length of Last Word

class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        if (s.length() == 0) {
            return 0;
        }
        String[] words = s.split(" ");
        int res = 0;
        return words[words.length - 1].length();
    }
}